XamarinCommunityToolkit/Actions/Snackbar/iOS/Extensions/UIViewExtensions.ios.cs

```diff
@@ -0,0 +1,77 @@
+﻿using UIKit;
+
+namespace Xamarin.CommunityToolkit.Actions.Snackbar.iOS.Extensions
+{
+	static class UIViewExtensions
+	{
+		public static NSLayoutYAxisAnchor SafeBottomAnchor(this UIView view)
+		{
+			return UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
+				? view.SafeAreaLayoutGuide.BottomAnchor
```

Comment:
Would you mind to update this file to follow the code-style with lambda-method-body for one line methods.

```
=> UIDevice.CurrentDevice.CheckSystemVersion(11, 0)
	? view.SafeAreaLayoutGuide.BottomAnchor
	: view.BottomAnchor;
```

Also, it makes sense to make a separate property **IsIos11OrHigher** and use it